### PR TITLE
dooble: bump version

### PR DESCRIPTION
**Pull request by kenmays**
dooble-20221115
Dooble 2022.11.15 on Haiku hrev56645 x64

### CODE REVIEW

File: www-client/dooble/dooble-1.56f.recipe [hunk 1]
```diff
@@ -6,7 +6,7 @@ manager, and many others."
 HOMEPAGE="https://textbrowser.github.io/dooble/"
 COPYRIGHT="Dooble"
 LICENSE="BSD (3-clause)"
-REVISION="4"
+REVISION="5"
```
Comment (Begasus) [inline www-client/dooble/dooble-1.56f.recipe:9]: Haven't checked a build, but why keep the older version if the newer version is fine?

Comment (kenmays) [reply www-client/dooble/dooble-1.56f.recipe:9]: Meant to delete it....

**Comment by Begasus** [comment]
The patchset isn't needed anymore here? pings @threedeyes

**Comment by kenmays** [comment]
No. There was some upstream integration so seems to build and work well (as is) for me (in comparison to Falkon).

**Comment by kenmays** [comment]
@Begasus - The patches are no longer valid.

**Comment by Begasus** [comment]
Merged, keep an eye out on the buildmaster and be sure to check the new package once build, thanks.

**Pull request merged**